packages/design/tailwind/css/components/select.css

```diff
@@ -77,12 +77,12 @@
 }
 
 .gi-select-option-item {
-  @apply gi-mx-1 
+  @apply gi-mx-1 gi-mb-2 last:gi-mb-0  
```

Comment:
Would the padding work better for this case?
It might also require a `first:gi-mb-0` (not sure if it's the proper syntax).